src/Components/Locations.js

```diff
@@ -0,0 +1,14 @@
+/** @format */
+
+import React from 'react';
+
+const Locations = () => {
+	return (
+		<div className='locations'>
+			<h2>List of Locations</h2>
+			<button type='submit'>Show Locations</button>
+		</div>
+	);
+};
+
```

Comment:
1. Fetch data
2. sort this dat based on button clicks;
3. each button needs a 'value' - climate, name 
4. we need an onlCick that captures that `value`
5. pass that value to a function that uses `.sort(value)`
6. new array from state - call sort w/ value
7. in the sort compare a[key] to b[key]
8. update our state